pyproject.toml

```diff
@@ -23,6 +23,8 @@ dependencies = [
     "netcdf4",
     "shapely",
     "pyproj",
+    "xarray",
+    "ocean_model_grid_generator@git+https://github.com/nikizadehgfdl/ocean_model_grid_generator@790069b31f9791864ccd514a2b8f53f385a0452e"
```

Comment:
It seems this is only used for tests. Could you move it to `[project.optional-dependencies]`?

Reply:
The same for xarray.

Reply:
Thanks done :)